docs/anchoring-assets/anchor-platform/sep31/integration.mdx

```diff
@@ -15,6 +18,258 @@ The following may also be required depending on your use case:
 - [`GET /unique_address`][get-unique-address] if your business uses a custody service for on-chain assets
 - [`DELETE /customer`][delete-customer] if your business wants or is required to allow senders to request deletion of customer data
 
+## Create a Business Server
+
+First, lets create a business server and add it to our docker compose file.
+
+<CodeExample>
+
+```yaml
+version: "3.8"
+
+services:
+  platform:
+    image: stellar/anchor-platform:2.0.0-rc.2
+    command: --sep-server
+    ports:
+      - "8080:8080"
+    env_file:
+      - ./dev.env
+    volumes:
+      - ./config:/home
+  server:
```

Comment:
Can you add a paragraph (roughly):

```
For local development, you can start your business server on your local machine. To reference your implementation, update configuration replacing `server` to `host.docker.internal`. See [docker documentation](https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host) for more information

:::tip

For Linux machines, it's required to use host-gateway. For each services, following configuration should be added:
services:
  service:
    extra_hosts:
      - host.docker.internal:host-gateway
      
See [github](https://github.com/docker/for-linux/issues/264#issuecomment-965465879) for more info

:::
```

Reply:
I'm not sure this is really in scope. They can choose to run their business server outside docker, but if they choose to go that route, they can figure out how to connect their anchor platform instance to it.

Reply:
I've been running my business server within docker and just have it auto-reload when the code changes.

Reply:
If you think so. Not all languages support hot reload